docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+
+## Step 2/4. Create Bot User
+
+Before you create a bot user, you need to determine which role you want to assign to the bot user. You can use `tctl` to examine what roles exist on your system.
+
+```bash
+$ tctl get roles --format=text
+Role    Allowed to login as                           Node Labels Access to resources                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
+------- --------------------------------------------- ----------- -----------------------------------------
+access  {{internal.logins}}                           <all nodes> event:list,read,session:read,list                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+editor                                                            user:list,create,read,update,delete,...
+```
+
+The above output is from a fresh install of Teleport with the default roles, your cluster may have additional roles. In our example, let’s assume you want to give the bot `access` role to allow it to connect to SSH servers within your cluster.
+
+```bash
+$ tctl bots add robot --roles=access
+```
+
+## Step 3/4. Start Machine ID
+
+Now start Machine ID using the `tbot` binary.
+
+```
```

Comment:
Is it worth adding a note here that this is using "token-based" joining and that we also support IAM join?